README.md

```diff
@@ -1,2 +1,183 @@
+
+In order to run this project you need :arrow_down: :
+
+- Check the Ruby version or if you have at all by running `ruby -v` if it returns  `ruby 3.1.2p20 (2022-04-12 revision 4491bb740a)` or similar, you have Ruby installed.
+
+* Ruby version used `ruby 3.1.2`
+
+
+### Setup 🛠️
+
+Clone this repository to your desired folder:
+
+
+- Clone the repository using the GitHub this link [🔗](https://github.com/karlavdelgadof/PlanB-app.git)
+- `cd Plan-app`
+
+### Install 📥
+
+To get this project running correctly, please run the following command to install all necessary dependencies: 
+
+- Run `bundle install`
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `rails server`
+- Open Browser `http://localhost:3000/`
+- Ensure you have PostgreSQL installed before running the application`psql --version`.
+- Run `rails db:setup`.
+- Create a [.envs](https://medium.com/geekculture/postgresql-rails-and-macos-16248ddcc8ba) file using the link to host postgres credentials.
+
+### Run tests ⛔
+
+To run tests:
+
+- Ensure you have `RSpec` installed: running bundle install will install it since it is included in the `Gemfile`.
+ - To run a specific test:
+ - Run `rspec spec/models<name_of_spec_file>`
+ - To run all tests:
+ - Run `rspec spec`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 :writing_hand: Authors <a name="authors"></a>
+
+:woman_technologist:  **Karla Delgado** :writing_hand: 
+
+- GitHub: [@karlavdelgadof](https://github.com/karlavdelgadof)
+- Twitter: [@karlavdelgadof](https://twitter.com/karlavdelgadof)
+- LinkedIn: [Karla Delgado](https://www.linkedin.com/in/karla-delgado-613a32239/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project or find it useful/interesting, please make sure you give ⭐️ a since this will make it easily accesible for you too!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/Gregoire Vella/">Creative Commons Attribution 4.0 International License</a>.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
OPTIONAL
- [ ] Kindly fix the broken link.
- [ ] Also make sure the readme file has not empty section.